app/uk/gov/hmrc/economiccrimelevyreturns/views/AmountDueView.scala.html

```diff
@@ -88,7 +88,18 @@ <h2 class="govuk-heading-m">@messages("amountDue.bands")</h2>
 
         <p class="govuk-body">@messages("amountDue.bands.p1")</p>
         <p class="govuk-body">@messages("amountDue.bands.p2")</p>
-        <p class="govuk-body">@messages("amountDue.bands.p3")</p>
+
+        @if(calculatedLiability.amountDue.amount == 0) {
+
+            <p class="govuk-body">@messages("amountDue.bands.nil.p1")</p>
+            <p class="govuk-body">@messages("amountDue.bands.nil.p2")</p>
+            <p class="govuk-body">@messages("amountDue.bands.nil.p3")</p>
+
+        } else {
+
+            <p class="govuk-body">@messages("amountDue.bands.p3")</p>
```

Comment:
This should only be displayed if this is not a nil return AND the user has not carried out AML regulated activity for the full FY (the content now also needs to by dynamic based on the AP length and AML activity period).

Reply:
This message is not dynamic in the present code. There is nothing in the ticket to make it dynamic!

Reply:
The ticket describes this under the bold heading "Dynamic change to "amount due page" if user is not regulated for Full FY". In addition the content in Confluence has alternative content for a full AML regulated financial year versus one that is not.